controller.go

```diff
@@ -617,6 +617,32 @@ func getHostConfig(c *Controller, tunnel *inletsv1alpha1.Tunnel) provision.Basic
 			UserData:   userData,
 			Additional: map[string]string{},
 		}
+
+	case "azure":
+		// Ubuntu images can be found here https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage#list-popular-images
+		// An image includes more than one property, it has publisher, offer, sku and version.
+		// So they have to be in "Additional" instead of just "OS".
+
+		inletsPort := inletsOSSControlPort
+
+		if c.infraConfig.UsePro() {
+			inletsPort = inletsPROControlPort
+		}
+		host = provision.BasicHost{
+			Name:     tunnel.Name,
+			OS:       "Additional.imageOffer",
+			Plan:     "Standard_B1ls",
```

Comment:
Was that the same plan as in the inletsctl version? Is there a cheaper option, or is this the best one for now?

Reply:
It is same as what we have in inletsctl. It is the cheapest plan: https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-b-series-burstable